test/rewrite_test.go

```diff
@@ -72,15 +110,43 @@ func testEdns0(t *testing.T, server string) {
 	if o == nil || len(o.Option) == 0 {
 		t.Error("Expected EDNS0 options but got none")
 	} else {
+		if len(o.Option) != 1 {
+			t.Errorf("Expected only one EDNS0 options but got %d", len(o.Option))
+		}
 		if e, ok := o.Option[0].(*dns.EDNS0_LOCAL); ok {
 			if e.Code != 0xffee {
 				t.Errorf("Expected EDNS_LOCAL code 0xffee but got %x", e.Code)
 			}
-			if !bytes.Equal(e.Data, []byte("hello-world")) {
-				t.Errorf("Expected EDNS_LOCAL data 'hello-world' but got %q", e.Data)
+			if !bytes.Equal(e.Data, []byte("initialization")) {
+				t.Errorf("Expected EDNS_LOCAL data 'initialization' but got %q", e.Data)
 			}
 		} else {
-			t.Errorf("Expected EDNS0_LOCAL but got %v", o.Option[0])
+			t.Errorf("Expected EDNS_COOKIE but got %v", o.Option[0])
```

Comment:
Why EDNS_COOKIE ?

Reply:
error of test of code. forgot to revert back.